libopenage/renderer/opengl/render_pass.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017-2019 the openage authors. See copying.md for legal info.
+
+#include "render_pass.h"
+
+namespace openage {
+namespace renderer {
+namespace opengl {
+
+GlRenderPass::GlRenderPass(const std::vector<Renderable>& renderables, RenderTarget const* target)
+        :RenderPass(renderables, target), is_optimised(false) {}
+
+std::vector<Renderable> GlRenderPass::get_renderables() const {
+    return this->renderables;
+}
+
+void GlRenderPass::set_renderables(std::vector<Renderable> renderables) {
+    this->renderables = renderables;
+    this->is_optimised = false;
+}
+
+bool GlRenderPass::get_is_optimised() const {
+    return this->is_optimised;
+}
+
+void GlRenderPass::set_is_optimised(bool flag) {
+    this->is_optimised = flag;
```

Comment:
tab indentation please

Reply:
ah I thought make check all checks for all this... will get to it right away